README.md

```diff
@@ -1,5 +1,8 @@
-# Development stop
-As of October 2016 the development of the MyBB Merge System has been stopped in favour of the development of MyBB 2.0. [read more on our blog](https://blog.mybb.com/2016/10/17/mybb-1-8-8-merge-system-1-8-8-release/)
+# Development continues
```

Comment:
I think we could maybe just remove this whole section?

Reply:
OK, updated. Maybe that's enough for now, after 1823 release we can make this file rich.